test/extended/kubevirt/util.go

```diff
@@ -0,0 +1,317 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/origin/test/extended/networking"
+	exutil "github.com/openshift/origin/test/extended/util"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/storage/names"
+	k8sclient "k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+	"k8s.io/kubernetes/test/e2e/framework/pod"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type NodeType int
+
+type IPFamily string
+
+const (
+	// Initial pod start can be delayed O(minutes) by slow docker pulls
+	// TODO: Make this 30 seconds once #4566 is resolved.
+	podStartTimeout = 5 * time.Minute
+
+	// How often to poll pods and nodes.
+	poll = 5 * time.Second
+
+	// How wide to print pod names, by default. Useful for aligning printing to
+	// quickly scan through output.
+	podPrintWidth = 55
+
+	// Indicator for same or different node
+	SAME_NODE      NodeType = iota
+	DIFFERENT_NODE NodeType = iota
```

Comment:
you're not actually using all of these constants that you copied over

Reply:
yes, right.
eventually I used the consts directly from the networking package.